tests/integration/test_with_flow.py

```diff
@@ -28,7 +28,7 @@ def test_integration(request_size: int):
             return_results=True,
         )
 
-    assert sum(len(resp_batch.docs) for resp_batch in resp) == 50
+    assert len(resp) == 50
```

Comment:
I can not select it here, but in the line number 23., we don't need the `return_results=True` param. It should by default return the `DocumentArray`